.github/workflows/release.yml

```diff
@@ -20,3 +20,6 @@ jobs:
       with:
         version: latest
         args: release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
GitHub provides automatic token authentication, see https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication.

I think that you'd need something like this:

```
permissions:
  contents: write
```

Reply:
There is an example here: https://github.com/goreleaser/goreleaser-action?tab=readme-ov-file#workflow.